server/Assignments_filler/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cachetools==4.2.1
+certifi==2020.12.5
+chardet==4.0.0
+google-api-core==1.26.3
+google-api-python-client==2.2.0
+google-auth==1.29.0
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.4
+googleapis-common-protos==1.53.0
+httplib2==0.19.1
+idna==2.10
+mysql-connector-python==8.0.23
+oauthlib==3.1.0
+packaging==20.9
+protobuf==3.15.8
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pyparsing==2.4.7
+pytz==2021.1
+requests==2.25.1
+requests-oauthlib==1.3.0
+rsa==4.7.2
+six==1.15.0
+uritemplate==3.0.1
+urllib3==1.26.4
+
+Also requires "credentials.json" from the google API project page.
```

Comment:
This file is generated with `pip freeze > requirements.txt` and defines specific requirements for a virtual environment so it can always be ran with same exact libraries.

Next time requirements change or get updated, this line will be gone. Put it into a `README.md` instead along with all other necessary instructions.

Reply:
Not resolved yet.

Reply:
Stop marking this as solved, it was not. Regenerate the file as I wrote above.